<ui version="4.0" >
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Path</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>designer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDesignerPath" />
      </item>
      <item>
       <widget class="QPushButton" name="bDesignerPath" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Interface</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Icon size:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbIconSize" >
          <item>
           <property name="text" >
            <string>12 x 12</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>16 x 16</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>24 x 24</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>32 x 32</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>48 x 48</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="drawSelection" >
        <property name="layoutDirection" >
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Draw selection border around items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
